uwblueprint/ccbc
# [Feat] Add background image to review base page

## Notion ticket link

[Add bg image to review base page](https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=fcfa0c91dbdf492780c0cc235b9e8883)



## Implementation description
* Added background image and styling 



## Steps to test
1. Go to http://localhost:3000/reviews/{ID} where ID is a valid review id number. Eg. http://localhost:3000/reviews/2. Verify that background image is loaded properly and sized properly too and matches figma designs.
2. Go on mobile view and verify bg image isn't there.


## What should reviewers focus on?
* 


## Checklist
